doc/api/vm.md

```diff
@@ -1908,6 +1908,7 @@ has the following signature:
 * `importAttributes` {Object} The `"with"` value passed to the
   [`optionsExpression`][] optional parameter, or an empty object if no value was
   provided.
+* `phase` {string} The phase of the dynamic import (`"source"` or `"evaluation"`).
 * Returns: {Module Namespace Object|vm.Module} Returning a `vm.Module` is
```

Comment:
Given that `WebAssembly.Module` is the only type of `ModuleSource`, we should document it here.

```suggestion
* Returns: {Module Namespace Object|vm.Module|WebAssembly.Module} Returning a `vm.Module` is
```

Reply:
This design currently only supports returning a `vm.Module` and not a `WebAssembly.Module`. We can ideally follow up with the `HostResolveModuleSourceModuleRecord` based associating as per https://github.com/tc39/proposal-source-phase-imports/pull/70 in due course, but for now these aspects aren't yet integrated.